sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -4,6 +4,7 @@
 """
 
 try:
+    import time
     import threading
```

Comment:
Sort these alphabetically. Also suggest removing the `try/except` statement. It is a legacy tradition in the code base which doesn't really serve much purpose beside adding a few words to the stack trace.

Reply:
fixed